google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -424,6 +424,8 @@
     <applicationService serviceImplementation="com.google.cloud.tools.intellij.csr.CloudRepositoryService"/>
     <applicationService serviceImplementation="com.google.cloud.tools.intellij.apis.CloudLibrariesService"/>
     <applicationService serviceImplementation="com.google.cloud.tools.intellij.apis.CloudApiMavenService"/>
+    <applicationService serviceInterface="com.google.cloud.tools.intellij.apis.ServiceAccountKeyDialogService"
```

Comment:
I a bit confused by this - the serviceInterface and serviceImplementation are pointing to the same class?

Reply:
in other words - why not just have a serviceImplementation only in the xml

Reply:
Oh didn't know I could do that. Will do.